.gitlab-ci.yml

```diff
@@ -23,3 +29,57 @@ linux-test:
     - yarn lint
   tags:
     - linux-docker
+
+#### stage:                        dockerize
+
+.dockerize_init:                   &dockerize_init
+  variables:
+    DOCKER_DRIVER: overlay2
+    DOCKER_HOST: tcp://localhost:2375
+  tags:
+    - kubernetes-parity-build
+  image: docker:git
+  services:
+    - docker:dind
+  before_script:
+    - POD_NAME=$(echo ${CI_JOB_NAME} | sed -E 's/^[[:alnum:]:]+-//')
+    - export DOCKER_IMAGE="${CI_REGISTRY}/${KUBE_NAMESPACE}-${POD_NAME}"
+    - export DOCKER_IMAGE_FULL_NAME=$DOCKER_IMAGE:$DOCKER_TAG
+    - echo $DOCKER_IMAGE_FULL_NAME
+    - echo "$DOCKER_REGISTRY_JSON" | docker login -u _json_key --password-stdin https://gcr.io
+
+dockerize-frontend:
```

Comment:
We've been using Netlify to deploy the frontend for the simplicity but with this available I think we can drop Netlify.

What do you think @amaurymartiny ?

Reply:
Nice, cool! Yes, let's drop Netlify.

Reply:
Although, the gitlab setup won't deploy a static frontend on each PR. Do we want to keep that?